packages/test-sync/src/androidAndroidTest/kotlin/io/realm/kotlin/test/mongodb/shared/ProgressListenerTests.kt

```diff
@@ -58,7 +58,7 @@ import kotlin.time.Duration
 import kotlin.time.Duration.Companion.seconds
 
 private const val TEST_SIZE = 500
-private val TIMEOUT = 10.seconds
+private val TIMEOUT = 15.seconds
```

Comment:
The notifier is now initialized on first subscription instead of when the realm is opened. This changed the timing a bit as the notifier initialization is now part of the monitored time interval.